lib/puppetlabs_spec_helper/util.rb

```diff
@@ -0,0 +1,9 @@
+require 'puppet'
+
+module PuppetlabsSpecHelper; end
+
+module PuppetlabsSpecHelper::Util
+  def puppet_3_or_older?
+    Puppet.version.split('.').first.to_i <= 3
```

Comment:
Any reason you didn't use the semver stuff from `Gem::Version` ? e.g.

``` ruby
Gem::Version.new(Puppet.version) <  Gem::Version.new('4.0.0')
```

Reply:
This is a bit lighter weight as I'm only concerned about the major version here which can be handled by a simple integer comparison, but I'm fine with updating to use `Gem::Version`s instead.